index.js

```diff
@@ -18,7 +19,7 @@ class Equivalency {
     return this;
   }
 
-  equivalent(s1, s2) {
+  equivalent(s1, s2, options = null) {
```

Comment:
You can defer this to the PR where you update the docs overall, but just noting that I think it'd be appropriate to document this function using jsdoc-style comments, since it is the main entrypoint for doing a comparison and now it takes more than just the two strings to be compared. That looks like this (you can search for examples of it in our codebase):

```js
/**
 * Function that does something useful.
 *
 * @param {number|string} id                Usefulness id.
 * @param {Object}        options           Options hash.
 * @param {bool}          options.compress  Whether or not to compress usefulness.
 *
 * @return {Object} Returns an object with the following top-level 
 *                  properties:
 *                  - prop1
 *                  - prop2
 */
function doThing() { ...
...
```